app/controllers/api/restful_controller.rb

```diff
@@ -48,18 +48,29 @@ def public_records
 
   def respond_with_resource(scope: default_scope, serializer: resource_serializer, root: serializer_root)
     if resource.errors.empty?
-      respond_with_collection resource_options(scope: scope, serializer: serializer, root: root)
+      respond_with_collection scope: scope, serializer: serializer, root: root
     else
       respond_with_errors
     end
   end
 
-  def resource_options(scope:, serializer:, root:)
-    if @event.is_a?(Event)
-      { resources: [@event], scope: scope, serializer: EventSerializer, root: :events }
+  def respond_with_collection(scope: default_scope, serializer: resource_serializer, root: serializer_root)
```

Comment:
Okay, I've highlighted the 'serializing events vs serializing models' here a bit more, although it's the same logic.

- If @event is an Event, then we're serializing out events.
- The way we do this, is wrap up the events into an EventCollection, then serialize that
- An EventCollection serializer is real simple; it `has_many :events`
- The reason we do this is that it allows the individual events to choose their own serializer (A new_motion event can serialize a motion, a discussion_edited can serialize a version, etc.) This simplifies the logic tremendously (check out `app/serializers/events/*_serializer.rb`), and there's no way to do this with ActiveModel::ArraySerializer, which requires a single `each_serializer` option to specify one serializer for the whole array.